.github/pull_request_template.md

```diff
@@ -0,0 +1,87 @@
+# Pull Request Title
+## Fixes \#(your issue number)
+
+<br>
```

Comment:
Suggest to remove HTML syntax and retain pure markdown

Reply:
There are a number of such places to do this.

Reply:
Happy to swap but <br /> or adding a &nbsp; is the only thing that works in my editor for adding spaces between headings. Appreciate doing something like this is outside CommonMark (the github one) --- is this bad practice anyway? Google is ambiguous, I'd love to get your view. For now I'll drop the extra line break and remove html tags